pkg/driver/sanity_test.go

```diff
@@ -309,13 +321,48 @@ func (f *fakeMounter) GetDeviceName(mountPath string) (string, int, error) {
 }
 
 func (f *fakeMounter) MakeFile(pathname string) error {
+	file, err := os.OpenFile(pathname, os.O_CREATE, os.FileMode(0644))
```

Comment:
I know we talked about this offline, but I want to note it publicly. 

I am not quite sure why we have a different mounter for sanity tests. We should use the code from the driver itself, otherwise we're not really testing the "real" code. 

/cc @wongma7 @gnufied

Reply:
I think it's fine, I'm not too familiar with sanity but seems mocking the mount is common pattern. https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/blob/2014365ea7b3405e024be1d9552aa30d44da4a8b/pkg/mount-manager/fake-safe-mounter.go#L28

I think the sanity test is still testing everything in the driver code surrounding wheerever it actually tries to call "Mount/Statfs".

Reply:
Should be fine as long as we don't modify the mount, but if we do that we risk silently failing sanity tests.

@AndyXiangLi can you add a note to the actual mount code? That it's mirrored here and any modification should be applied here too. Not ideal, but I'd feel safer.

Reply:
@ayberk Sure will do that, and as we talked off line, I think it would be better if we can extract those three functions out of mounter in the future.